cloudsync/providers/file.py

```diff
@@ -72,7 +76,7 @@ def casedpath(path):
     if is_osx():
         url = NSURL.fileURLWithPath_(path)  # will be None if path doesn't exist
         if not url:
-            return path
+            return None
```

Comment:
You changed this to return None, which is all well and good, but the code below still returns path if the file isn't found

Reply:
does it matter?

Reply:
i get it now, fixed